docs/robots/husky/troubleshooting_husky.mdx

```diff
@@ -214,3 +214,41 @@ Verify the upstart logs, `/var/log/upstart/husky_core.log`, to see if there are
 
 1.  Check that the sensor's User Power connector has the correct voltage.
 2.  If the voltage is correct, then review the [Troubleshooting section for the related sensor](/docs/sensors).
+>>>>>>> development
+
+
+TODO: Check if these are incorporated:
+
+**The blue light around the power button does not display when pressing the power button.**
+
+1.  Ensure that the battery is charged and correctly connected.
+2.  Use a multi-meter to verify the voltage on the battery terminals.
+
+**_Comm Status_ light stays red or flickers.**
+
+1.  Your program may not be sending motion commands faster than 10 Hz.
+2.  If there is a poor quality connection, some commands may be lost, so it may be necessary to increase the frequency of commands sent.
+3.  The maximum frequency commands can be sent is 50 Hz.
+
+**_E-stop Status_ light illuminated**
```

Comment:
Suggest wee change this to 'Stop light'. This is for consistency, and to deemphasize emergency.

Reply:
Done